backend/src/main/java/com/woowacourse/thankoo/alarm/application/strategy/coupon/CouponMealMessageFormStrategy.java

```diff
@@ -30,7 +31,8 @@ public Message createFormat(final Alarm alarm) {
         return Message.builder()
                 .email(receiverEmails)
                 .title(MEAL_PRETEXT)
-                .titleLink(alarmLinkGenerator.getRootUrl())
+                .titleLink(alarmLinkGenerator.createUrl(
+                        MessageFormat.format(ACCEPT_TITLE_LINK, alarm.getOrganizationId())))
```

Comment:
```suggestion
String url = alarmLinkGenerator.createUrl(MessageFormat.format(ACCEPT_TITLE_LINK, alarm.getOrganizationId()))
                .titleLink(url)
```
이렇게 분리하는게 더 깔끔하지 않을까?